web/thesauruses/rust/1/lists.json

```diff
@@ -0,0 +1,288 @@
+    },
+    "erase_element_from_mutable_hashed_list": {
+      "code": "hashset.remove(\"foo\");"
+    },
+    "delete_mutable_hashed_list": {
+      "code": "hashset.clear();"
+    },
+    "name_of_immutable_hashed_list": {
+      "code": "HashSet<T>",
+      "comment": "Rust uses the 'mut' keyword to determine if a variable is mutable"
+    },
+    "create_a_immutable_hashed_list": {
+      "code": "let hashset = HashSet::new();"
+    },
+    "insert_element_to_immutable_hashed_list": {
+      "not-implemented": "true"
+    },
+    "erase_element_from_immutable_hashed_list": {
+      "not-implemented": "true"
+    },
+    "delete_immutable_hashed_list": {
+      "not-implemented": "true"
+    },
+    "create_a_mutable_set": {
+      "code": "let mut hashmap: HashMap<TKey, TValue> = HashMap::new();"
+    },
+    "get_key_from_mutable_set": {
+      "not-implemented": "true"
+    },
+    "get_value_from_mutable_set": {
+      "code": "hashmap.get(\"foo\");"
+    },
+    "get_all_keys_from_mutable_set": {
+      "code": "hashmap.keys();",
+      "comment": "Note this is in an arbitrary order"
+    },
+    "get_all_values_from_mutable_set": {
+      "code": "hashmap.values();",
+      "comment": "Note this is in an arbitrary order"
+    },
+    "swap_key_and_value_in_mutable_set": {
+      "not-implemented": "true"
+    },
+    "delete_mutable_set": {
+      "code": "hashmap.clear();"
+    },
+    "create_a_immutable_set": {
+      "code": "let hashmap: HashMap<TKey, TValue> = HashMap::new();"
+    },
+    "get_key_from_immutable_set": {
+      "not-implemented": "true"
+    },
+    "get_value_from_immutable_set": {
+      "code": "hashmap.get(\"foo\");"
+    },
+    "get_all_keys_from_immutable_set": {
+      "code": "hashmap.keys();",
+      "comment": "Note this is in an arbitrary order"
+    },
+    "get_all_values_from_immutable_set": {
+      "code": "hashmap.values();",
+      "comment": "Note this is in an arbitrary order"
+    },
+    "swap_key_and_value_in_immutable_set": {
+      "not-implemented": "true"
+    },
+    "delete_immutable_set": {
+      "not-implemented": "true"
+    },
+    "find_element_by_value": {
+      "code": "vec.iter().position(|&r| r == \"foo\").unwrap();"
+    },
+    "find_minimum_element": {
+      "code": "vec.iter().min();"
+    },
+    "find_maximum_element": {
+      "code": "vec.iter().max();"
+    },
+    "convert_list_to_string": {
+      "code": "vec.join(\" \");"
+    },
+    "concatenate_two_lists": {
+      "code": "vec.append(secondVec);"
+    },
+    "split_list_at_index": {
+      "code": "vec.split_off(index);"
+    },
+    "split_list_at_value": {
+      "not-implemented": true
+    },
+    "duplicate_a_list": {
+      "code": "let vec2 = vec.clone();"
+    },
+    "duplicate_subset_of_list": {
+      "not-implemented": true
+    },
+    "get_list_length": {
+      "code": "vec.len();"
+    },
+    "resize_list": {
+      "code": "vec.resize(new_len, T);"
```

Comment:
Capitalized identifiers like `T` usually denote a type in rust, I'd rather use somethig like `item`.